src/AspNetCoreModuleV2/CommonLib/hostfxr_utility.cpp

```diff
@@ -137,29 +138,49 @@ HOSTFXR_UTILITY::IsDotnetExecutable(const std::filesystem::path & dotnetPath)
 }
 
 void
-HOSTFXR_UTILITY::ParseHostfxrArguments(
+HOSTFXR_UTILITY::AppendArguments(
     const std::wstring &applicationArguments,
-    const fs::path     &applicationExePath,
     const fs::path     &applicationPhysicalPath,
     std::vector<std::wstring> &arguments,
     bool expandDllPaths
 )
 {
-    LOG_INFOF("Resolving hostfxr_main arguments application: '%S' arguments: '%S' path: %S", applicationExePath.c_str(), applicationArguments.c_str(), applicationPhysicalPath.c_str());
-
-    arguments = std::vector<std::wstring>();
-    arguments.push_back(applicationExePath);
+    // don't throw while trying to expand arguments
+    std::error_code ec;
```

Comment:
Do we really need to? What errors will is_regular_file throw that are considered non fatal?

Reply:
arguments might contain invalid path characters. Tests fail without this.